recipes/install-nodejs.rb

```diff
@@ -0,0 +1,28 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-nodejs
+
+clear_cache = node.fetch(:clear_npm_cache, false)
```

Comment:
We can force clear the npm cache through the "clear_npm_cache" variable in a stack's `custom_json` if we need to. Doesn't look like it's needed, but just in case.